modules/apps/commerce/commerce-product-options-web/src/main/resources/META-INF/resources/js/ProductOptions.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+// import slugify from 'commerce-frontend-js/utilities/slugify'
+
+import {debounce} from 'frontend-js-web';
+
+
+export default function ({namespace}) {
+	console.log('cucu')
+	var form = document.getElementById('#'+namespace + 'fm');
+
+	var keyInput = form.querySelector('#'+namespace+ 'key');
+	var nameInput = form.querySelector('#'+namespace + 'name');
+
+	var handleOnNameInput = function (event) {
+		keyInput.value = slugify.default(nameInput.value);
+	};
+
+	nameInput.addEventListener('input', debounce.(handleOnNameInput, 200));
```

Comment:
Copying from https://github.com/andrea-ale-sbarra/liferay-portal/pull/4/files#r549744036 since we did this almost simultaneously 😂 

This is a syntax error since `debounce` is a function. This should be just `debounce(...)`. As it is, the build shows the following error:

> { SyntaxError: /Users/jose.balsas/dev/liferay/portal/master/portal/modules/apps/commerce/commerce-product-options-web/src/main/resources/META-INF/resources/js/ProductOptions.js: Unexpected token (31:46)
>
>  29 | 	};
>  30 | 
>  31 | 	nameInput.addEventListener('input', debounce.(handleOnNameInput, 200));

For some reason, our build finishes successfully, but the files aren't built properly so you're likely to see after this something like:

> Uncaught SyntaxError: Cannot use import statement outside a module

or 

>  A require() call has failed but no failure handler was provided.

These are nothing but red herrings distracting you from the fact that your build failed and produced a bad output. 

Correcting this and re-deploying I see that `debounce` is the expected function.

/cc @bryceosterhaus, @wincent, we might want to look into this. Is it possible we've somehow messed with the exit codes or failed to catch them and send them upwards to gradle? This case apparently throws an unhandled exception which might be falling through the cracks:

> (node:51583) UnhandledPromiseRejectionWarning: Error: Command /Users/jose.balsas/dev/liferay/portal/master/portal/modules/node_modules/@liferay/npm-scripts/node_modules/.bin/babel --no-babelrc --config-file /Users/jose.balsas/dev/liferay/portal/master/portal/modules/apps/commerce/commerce-product-options-web/build/npmscripts/tmp/.babelrc src/main/resources/META-INF/resources --out-dir build/node/packageRunBuild/resources --source-maps exited with code 1